trinity/rpc/modules/eth.py

```diff
@@ -173,6 +174,10 @@ async def blockNumber(self) -> str:
         num = self.chain.get_canonical_head().block_number
         return hex(num)
 
+    async def chainId(self) -> str:
+        chain_id = self.chain.chain_id
+        return to_hex(text=str(chain_id))
```

Comment:
If `chain_id` is an int here, then I think we can skip some conversion steps here with:
```suggestion
        return to_hex(chain_id)
```

Reply:
Ahh, yup I completely forgot that `int`s were also accepted as a default arg in these conversion functions.